### PR TITLE
Add pinterest share button.

### LINKED ISSUE #15: Pinterest share button (feature request)
Can we have a pinterest share button also?

Thank you!!

### PR DESCRIPTION
**Pull request by J-Siu**
Close #15

### CODE REVIEW

**Comment by J-Siu** [comment]
It is working on my site https://johnsiu.com/

File: layouts/partials/post_share.html [hunk 1]
```diff
@@ -35,5 +35,11 @@
 		{{ partial "svg/pocket.svg" (dict "class" "share__icon" ) }}
 	</a>
 	{{- end }}
+	{{- if .Param "share.pinterest" }}
+	<script type="text/javascript" async defer src="//assets.pinterest.com/js/pinit.js"></script>
+	<a class="share__link btn" title="{{ T "save-caption" }} Pinterest" href="https://www.pinterest.com/pin/create/button/" data-pin-do="buttonBookmark" data-pin-custom=true>
```
Comment (Vimux) [inline layouts/partials/post_share.html]: `onclick` action should be preserved too (see any other button for example)

**Comment by J-Siu** [comment]
Updated the pinterest button to be inline with others, no more external js  :)

**Comment by Vimux** [comment]
Do not forget to add Pinterest at the end of the share buttons section:

https://github.com/Vimux/Binario/blob/e72cb22fa33f864238b5bcee6c692588a1b6fd49/README.md#L305

**Comment by J-Siu** [comment]
> @J-Siu, do not forget to add Pinterest at the end of the share buttons section:
> 
> https://github.com/Vimux/Binario/blob/e72cb22fa33f864238b5bcee6c692588a1b6fd49/README.md#L305

Done :)

**Pull request merged**